src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy

```diff
@@ -78,7 +78,7 @@ class WaveContainerRecordTest extends Specification {
         container.timestamp == OffsetDateTime.parse(req.timestamp).toInstant()
         container.zoneId == OffsetDateTime.parse(req.timestamp).offset.id
         container.expiration == exp
-        container.fusionVersion == req.containerConfig.fusionVersion()
+        container.fusionVersion == req.containerConfig.fusionVersion().number
```

Comment:
Can `fusionVersion()` return `null`?

Reply:
then 
```suggestion
        container.fusionVersion == req.containerConfig.fusionVersion()?.number
```

then, it should be like this, no? ☝️

Reply:
sorry this is a test, where I already added fusion layer
https://github.com/seqeralabs/wave/blob/1f39d0bf8b1c68c6aa73d108c53d6bcd36627526/src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy#L43

Reply:
oops, ok